### ISSUE TITLE
Caching issues with testSib accidentals.sib

### ISSUE THREAD

**Issue by annplaksin** [music-encoding/sibmei#119]
I run into a problem with the sibmei tests consisting partially of the Sibelius internal test routines done with sib-test and the node,js-based testing environement.
There might be some caching issues that affects the export of the _accidentals.sib_ in the batch export that is made after the Sibelius interal tests are finished.

When I run the test runner, the content of the accidentals.mei doesn't match the content of the _accidentals.sib_. There aren't any accidentals but slurs in the export.
When I create a copy of the accidentals.sib (called 'accidentals2.sib), the export of the copy is fine. The export of the original is still weird.
As I tried to explore, if the problem is connected to the file itself or might be based on being the first in the batch export, I created a copy of the lyrics.sib called _aaa_lyrics.sib_. That export looks fine but the content of _accidentals.mei_ changed from the crappy slur stuff to the content of that aaa_lyrics.sib

That problem might occur with that test sib because it is the only file which is already used in the Sibelius internal tests.
I don't understand that behaviour completely.

Is there any way to fix this?

**Comment by th-we** [comment]
Did you check if you look at the correct file?  The "original" file is in the test folder and the one that the test routine is allowed to mess with is copied to the build folder.  Did you allow all files to be closed between test runs?  If you run into an issue like this, you can try deleting the MEI Export folder and just re-save a file in the source folder to trigger a re-build that will also re-copy the files.

**Comment by annplaksin** [comment]
Yes, I looked at the correct file in the build folder and it looks like the one in the test folder.
The problem is not in between test runs but in the same test run...
According to my various tests with renaming other test files in the build folder, adding a copy of the file and so on, I assume that it is just a bad idea to reuse a test file used in an internal test for another external test.
I originally tought about adding a new accidental test but then I decided to expand the existing test.

Maybe this is just something we have to keep in mind: Don't reuse test sibs... ;-)
